content/en/news/releases/1.5.x/announcing-1.5.6/index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Announcing Istio 1.5.6
+linktitle: 1.5.6
+subtitle: Patch Release
+description: Istio 1.5.6 patch release.
+publishdate: 2020-06-17
+release: 1.5.6
+aliases:
+    - /news/announcing-1.5.6
+test: n/a
+---
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.5.5 and Istio 1.5.6.
+
+{{< relnote >}}
+
+## Security
+
+- **Updated** Node.js and jQuery versions used in bookinfo.
+
+## Changes
+
+- **Fixed** Transfer-Encoding value case-sensitivity in Envoy ([Envoy's issue 10041](https://github.com/envoyproxy/envoy/issues/10041))
+- **Fixed** handling of user defined ingress gateway configuration ([Issue 23303](https://github.com/istio/istio/issues/23303))
+- **Fixed** Add `TCP MX ALPN` in `UpstreamTlsContext` for clusters that specify `http2_protocol_options` ([Issue 23907](https://github.com/istio/istio/issues/23907))
+- **Fixed** election lock for namespace configmap controller ([Issue 22463](https://github.com/istio/istio/issues/22463))
```

Comment:
I notice this issue is still open. Maybe replace it with  https://github.com/istio/istio/pull/22598.